prisma/migrations/20241124172312_rename_column/migration.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `shortUrl` on the `link` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[shortUrlCode]` on the table `Link` will be added. If there are existing duplicate values, this will fail.
```

Comment:
**issue**: Adding a unique constraint on `shortUrlCode` may fail if there are existing duplicate values. Consider checking for duplicates before applying this migration.